website/pages/en/billing.mdx

```diff
@@ -2,10 +2,32 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency through Banxa](#billing-on-banxa)
+- [Paying with GRT on Arbitrum](#billing-on-arbitrum)
+
+## Billing on Banxa
+
+Banxa enables you to bypass the need for an exchange and pay for your query fees using the fiat currency of your choice. The fiat currency will be converted to GRT, added to your billing balance on the billing contract, and used to pay for your query fees.
+
+There may be KYC requirements depending on the regulations in your country. For more information about KYC, please visit [Banxa's FAQ page](https://docs.banxa.com/docs/faqs).
+
+You can learn more about Banxa by reading their [documentation](https://docs.banxa.com/docs).
+
+### Paying for query fees with Banxa
+
+1. Select “Pay with Card” option in [Subgraph Studio](https://thegraph.com/studio/).
+2. Enter the amount of GRT to be added to your billing balance.
+3. Enter necessary banking information on Banxa including payment method & fiat currency of choice.
+4. Finish the transaction.
+
+Once the transaction is confirmed, your GRT will automatically be added to your billing balance on Arbitrum.
```

Comment:
"account balance"

Reply:
would say "the purchased GRT" or smth like that instead of "your" (if we are allowed to use "purchased")

Reply:
Good call. I think purchased works